src/agent/rustjail/src/container.rs

```diff
@@ -1540,14 +1545,27 @@ async fn execute_hook(logger: &Logger, h: &Hook, st: &OCIState) -> Result<()> {
             .unwrap();
         info!(logger, "child stdout: {}", out.as_str());
 
+        let mut err = String::new();
+        child
+            .stderr
+            .as_mut()
+            .unwrap()
+            .read_to_string(&mut err)
+            .await
+            .unwrap();
+        error!(logger, "child stderr: {}", err.as_str());
```

Comment:
Since below will also log an error message, here `error!` can be replaced with `info!`.

Reply:
Ok